lib/lte_link_control/lte_lc.c

```diff
@@ -1012,15 +997,14 @@ int lte_lc_edrx_req(bool enable)
 {
 	int err;
 	int actt[] = {AT_CEDRXS_ACTT_WB, AT_CEDRXS_ACTT_NB};
-	char req[25];
 
 	if (!enable) {
-		err = at_cmd_write(edrx_disable, NULL, 0, NULL);
+		err = nrf_modem_at_printf(edrx_disable);
 		if (err) {
-			LOG_ERR("Failed to disable eDRX, error: %d", err);
+			LOG_ERR("Failed to disable eDRX, reported error: %d", err);
 		}
 
-		return err;
+		return -EFAULT;
```

Comment:
Now this will always return -EFAULT after disabling eDRX (move `return -EFAULT` within `if (err)` and return zero here?).

Reply:
Heh, yeah, this was very wrong. Will fix